plugins/modules/ipasudorule.py

```diff
@@ -628,6 +655,10 @@ def main():
                         hostgroup_del = gen_intersection_list(
                             hostgroup, res_find.get("memberhost_hostgroup"))
 
+                    if hostmask is not None:
```

Comment:
hostmask can not be None due to line 372.

Reply:
fixed